builtin/credential/cert/path_config.go

```diff
@@ -0,0 +1,63 @@
+package cert
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config",
+		Fields: map[string]*framework.FieldSchema{
+			"verify_cert": &framework.FieldSchema{
```

Comment:
Two comments here: first, I wouldn't use the term `verify` since that sounds a lot like "verifying cert against CRL or OCSP" which is not what this is for and might be confusing. Second, I would invert this logic: have this be `disable_binding` with a default of `false`. Generally this is a good feature to default on, and that also allows a default configuration to do the right thing.